src/VisualStudio/CSharp/Impl/CSharpVSResources.resx

```diff
@@ -408,4 +408,16 @@
   <data name="NewLinesForBracesProperty" xml:space="preserve">
     <value>Place open brace on new line for properties, indexers, and events</value>
   </data>
+  <data name="QualifyMemberEventAccessWithThis" xml:space="preserve">
+    <value>Qualify member event access with 'this'</value>
```

Comment:
I kind of feel like the words "member" and "event" should be swapped. Maybe "member could just go?